tic-tac-toe.js

```diff
@@ -1,7 +1,221 @@
 function TicTacToe() {
+  this.board = [["","",""], ["","",""], ["","",""]];
+  this.count = 0;
+  }
 
-}
+  var checkSpot = function(col, row, sign , board) {
+    if (board[col][row] === sign) {
+      return true;
+    } else {return false;}
+  }
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.CheckRow = function(col, row, sign) {
+    if (row === 0) {
+      if (checkSpot(col, 1, sign, this.board) && (checkSpot(col,2, sign, this.board))) {
+        return true;
+    } else if ( row === 1) {
+      if (checkSpot(col, 0, sign, this.board) && (checkSpot(col,2, sign, this.board)))
+        return true;
+    } else {
+      if (checkSpot(col, 0, sign, this.board) && (checkSpot(col,1, sign, this.board)))
+        return true;
+    }
+  }
+};
+
+TicTacToe.prototype.CheckCol = function(col, row, sign) {
```

Comment:
Same comment with regards to generalizing this using the variable, I think in this case you can use `col` instead of `row` like above.